frontend_app_file/test/fixture/tracimLiveMessageData/commentTlm.js

```diff
@@ -0,0 +1,12 @@
+export const commentTlm = {
```

Comment:
this file should be in frontend_lib since it will be helpful feat every apps and frontend